my_enumerable.rb

```diff
@@ -0,0 +1,19 @@
+module MyEnumerable
+  def all?(&block)
+    result = true
+    each { |element| result = false unless block.call(element) }
+    result
+  end
+
+  def any?(&block)
+    result = false
+    each { |element| result = true if block.call(element) }
+    result
+  end
+
+  def filter(&block)
+    result = []
+    each { |element| result << item if block.call(element) }
```

Comment:
- [ ] There is something wrong with this line. The variable `item` is undefined. Please try to check this out.

See the screenshot of the error I got when testing your code.
![a1](https://user-images.githubusercontent.com/39523312/155822627-91d50335-4a3b-4f05-ac8b-a7803996fac3.png)